### PR TITLE
Support disabling targets

### PR DESCRIPTION
**Pull request by woodwoerk**
image

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205957048952290

### CODE REVIEW

**Comment by terotik** [comment]
Really nice solution for the coming soon. We got the German translation for that _werden später ergänzt_ (=will be added later)

**Review by terotik — APPROVED** [review]

**Review by terotik — CHANGES_REQUESTED** [review]
de Translation for coming-soon "werden später ergänzt"

File: public/locales/de/common.json [hunk 1]
```diff
@@ -82,5 +82,6 @@
   "table-unit": "Maßeinheit",
   "outcome-bar-summary-historical": "Die Summe der {{goalType}} im Jahr {{selectedYear}} war um {{selectedYearDifference}} höher als im Jahr {{nearestGoalYear}} ({{nearestGoalValue}}).",
   "outcome-bar-summary-forecast": "Mit dem gewählten Szenario {{scenarioName}} wird das Ziel von {{nearestGoalValue}} a im Jahr {{nearestGoalYear}} erreicht. Die Summe der {{goalType}} im Jahr {{selectedYear}} beträgt {{selectedYearValue}}.",
-  "outcome-tabs-label": "Anzeigeoptionen"
+  "outcome-tabs-label": "Anzeigeoptionen",
+  "coming-soon": "Bald verfügbar"
```
Comment (woodwoerk) [inline public/locales/de/common.json:86]:
```suggestion
  "coming-soon": "Werden später ergänzt"
```

**Review by terotik — APPROVED** [review]
Great thanks!

**Pull request merged**